compiler/aliases.nim

```diff
@@ -181,9 +181,16 @@ proc isPartOf*(a, b: PNode): TAnalysisResult =
       else: discard
     of nkObjConstr:
       result = arNo
-      for i in 1..<b.len:
+      for i in 1 .. <b.len:
```

Comment:
Isn't `<foo` deprecated?

Reply:
i.e isn't above same as `1 ..< b.len`? But that's the non-deprecated syntax?

Reply:
Ah, that's a typo. Thanks for checking :)